File: 프로그래머스/unrated/120831. 짝수의 합/짝수의 합.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    for(int i = 1; i <= n; i++) {
        if(i % 2 == 0) {
            answer += i;
        }
    }
    return answer;
}

int main() {
    int num1, num2;
    scanf("%d", &num1);
    
    num2 = solution(num1);
    printf("%d", num2);
    return 0;
}

File: 프로그래머스/0/120831. 짝수의 합/짝수의 합.java
class Solution {
    public int solution(int n) {
        int num1 = 0;
        for(int i = 1; i <= n; i++) {
            if(i % 2 == 0) {
                num1 += i;
            }
        }
        return num1;
    }
}